numba/tests/test_lists.py

```diff
@@ -692,8 +693,13 @@ def test_mul_error(self):
         pyfunc = list_mul
         cfunc = jit(nopython=True)(pyfunc)
         # Fail in malloc()
-        with self.assertRaises(MemoryError):
+        with self.assertRaises(MemoryError) as cm:
             cfunc(1, 2**58)
+        if sys.platform.startswith('darwin'):
+            libc = ct.CDLL('libc.dylib')
+            libc.printf("###Please ignore above error message i.e. \
```

Comment:
The wording here needs a bit of attention still. I think it should read:

> Please ignore the above error message i.e. can't allocate region. It is in fact the purpose of this test to request more memory than can be provided.

(note that some of the `the`'s have been moved!).

Reply:
Corrected the wordings as suggested